jenkins/hybrid_execution.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
it's better to make this script for preparing artifacts only and the actual caller place to run test command directly, 
or at least separate the func of artifacts preparation from the actual test call func.